package.json

```diff
@@ -4,8 +4,12 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "nodemon app.js"
+    "start": "node server/app.js",
+    "server": "nodemon node server/app.js",
+    "client": "npm run start --prefix client",
+    "dev": "concurrently \"npm run server\" \"npm run client\"",
```

Comment:
i don't get it what is this script doing ? cause `npm run client` is doing `npm run start` which also starts the server

Reply:
Again I need it so I can run it on development mode! look at "dev" script!